src/CarbonAware.CLI/src/Model/EmissionsDataDTO.cs

```diff
@@ -4,10 +4,12 @@ namespace CarbonAware.CLI.Model
 {
     public class EmissionsDataDTO
     {
+        private DateTimeOffset? _time;
+
         ///<example> eastus </example>
         public string? Location { get; set; }
         ///<example> 2022-09-19T17:30:00Z </example>   
-        public DateTimeOffset? Time { get; set; }
+        public DateTimeOffset? Time { get => _time?.ToUniversalTime(); set => _time = value; }
```

Comment:
Thanks for these changes. It looks more clear. One quick ask, given the fact this is cached, could you set it at setting time instead?
```suggestion
        public DateTimeOffset? Time { get => _time; set => _time = _value.ToUniversalTime(); }
```

Reply:
changed